client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -414,6 +450,10 @@ export default {
       return this.segment.relationships.comments?.data?.length || 0
     },
 
+    isAiTabActive () {
```

Comment:
not needed anymore.

Reply:
has been removed here https://github.com/demos-europe/demosplan-core/pull/1486/commits/564be88fd18b61f2f5f15b60268a73569a079c64